images/build/cross/Makefile

```diff
@@ -20,13 +20,12 @@ IMAGE=kube-cross
 
 TAG?=$(shell git describe --tags --always --dirty)
 CONFIG?=go1.14
-KUBE_CROSS_VERSION?=v1.14.2-3
+KUBE_CROSS_VERSION?=v1.14.2-4
 
 # Build args
 GO_VERSION?=1.14.2
-PROTOBUF_VERSION?=3.11.4
+PROTOBUF_VERSION?=3.9.1
```

Comment:
Oh, is this what it was before, but now we're just updating the Dockerfile to match? Maybe the Dockerfile var should become a dummy value (in a follow up maybe) and just use this to override.

Reply:
@cblecker -- Great point! Dropped in https://github.com/kubernetes/release/pull/1279/commits/2be1b2f6a7b738f6af7cb9ab99d8bbcb4e171b99.